internal/configs/node/network.go

```diff
@@ -39,6 +39,10 @@ const (
 )
 
 const (
+	// DefaultLocalListenIP ..
```

Comment:
I would suggest to finish these lines. Using ... is not good coding practice (I know the function name is already very elaborative, but would suggest to explain it in human language).

Reply:
Ill remove it then

Reply:
Hmm.. sure ill add some comments

Reply:
Sure. I definitely agree with you. A good code shall be self explanatory without any comments. But this is syntax required here :(